Source/WebKit/Shared/Extensions/wpe/WebExtensionUtilitiesWPE.cpp

```diff
@@ -40,9 +41,7 @@ Vector<double> availableScreenScales()
 
     GUniquePtr<GList> toplevels(wpe_toplevel_list());
```

Comment:
We shouldn't use wpe platform api unconditionally, we needs to check if the app is using it see `WKWPE::isUsingWPEPlatformAPI()`